public/scripts/app.js

```diff
@@ -10,6 +11,7 @@ function Project(argument){
   this.link = argument.link;
   this.img = argument.img;
 }
+//projects page//
```

Comment:
**TODO FOR LOST POINTS**

Now that you've gotten this wrapped in an IIFE, please indent your code properly.